indexd/src/lib.rs

```diff
@@ -1,16 +1,21 @@
 mod app_client;
+mod slabs;
+
+#[cfg(feature = "quic")]
+pub mod quic;
+#[cfg(feature = "quic")]
```

Comment:
I believe currently `use tokio::io::{AsyncReadExt, AsyncWriteExt};` would be unused if `quic` is not enabled.